rbx_types/src/basic_types.rs

```diff
@@ -166,6 +166,8 @@ impl CFrame {
     }
 }
 
+pub type OptionalCFrame = Option<CFrame>;
```

Comment:
I don't think we need this typedef because of how idiomatic and pervasive `Option` is.